rasa/utils/plotting.py

```diff
@@ -12,18 +12,34 @@
 logger = logging.getLogger(__name__)
 
 
-# At first, matplotlib will be initialized with default OS-specific available backend
-# if that didn't happen, we'll try to set it up manually
-if matplotlib.get_backend() is not None:
-    pass
-else:  # pragma: no cover
-    try:
-        # If the `tkinter` package is available, we can use the `TkAgg` backend
-        import tkinter
-
-        matplotlib.use("TkAgg")
-    except ImportError:
-        matplotlib.use("agg")
+def _fix_matplotlib_backend() -> None:
```

Comment:
fix is necessary due to issues of macos with tkiner on specific osx versions - the previous implementation was not working for me and matplot lib code failed with the tkagg error. this fixes that issue.